scripts/generate-manifest.js

```diff
@@ -29,12 +32,14 @@ const environmentIcons = {
 };
 
 const contentSecurityPolicyEnvironment = {
+  testing: `default-src 'none'; connect-src *; style-src 'unsafe-inline'; img-src 'self' data: https:; script-src 'self' 'wasm-unsafe-eval'; object-src 'none'; frame-src 'none'; frame-ancestors 'none';`,
   development:
     "script-src 'self' 'wasm-unsafe-eval'; object-src 'self'; frame-src 'none'; frame-ancestors 'none';",
   production: `default-src 'none'; connect-src *; style-src 'unsafe-inline'; img-src 'self' data: https:; script-src 'self' 'wasm-unsafe-eval'; object-src 'none'; frame-src 'none'; frame-ancestors 'none';`,
 };
 
```

Comment:
I'm curious, what is this doing? 🤔  

Do we need to change icons for tests? Can we add `-preview` here in `defaultIconEnvironment` or do we need to do it in `environmentIcons`

Reply:
I just copy pasted it from production, as I believed it need to be same as in production env 🤔